src/derived.rs

```diff
@@ -116,30 +111,6 @@ where
     }
 }
 
-pub enum VolatileValue {}
-impl<DB, Q> MemoizationPolicy<DB, Q> for VolatileValue
-where
-    Q: QueryFunction<DB>,
-    DB: Database,
-{
-    fn should_memoize_value(_key: &Q::Key) -> bool {
-        // Why memoize? Well, if the "volatile" value really is
-        // constantly changing, we still want to capture its value
-        // until the next revision is triggered and ensure it doesn't
-        // change -- otherwise the system gets into an inconsistent
-        // state where the same query reports back different values.
-        true
-    }
-
-    fn memoized_value_eq(_old_value: &Q::Value, _new_value: &Q::Value) -> bool {
-        false
-    }
-
-    fn should_track_inputs(_key: &Q::Key) -> bool {
```

Comment:
I think we can remove the `should_track_inputs` method from the trait entirely.

Reply:
indeed, fixed!